.github/workflows/main.yml

```diff
@@ -0,0 +1,134 @@
+name: Hypothesis CI
+
+on:
+  push:
+    branches: [ master , github-actions ]
```

Comment:
It is usefull to add `on: workflow_dispatch` to all actions. This will allow manual runs.

Reply:
Done - thanks!